### ISSUE TITLE
Unusual? behavior of the FanControl exe from the past 20'ish versions

### ISSUE THREAD

**Issue by FrozenApocalypse** [Rem0o/FanControl.Releases#2780]
Program operates as per normal, except for...

I don't know if this a bug or if this created by design, but upon every session of starting FanControl.exe it creates somewhere between 3-5 temp files in the windows/temp folder called TS_****.tmp (* appears to be written in Hex)
Temp files in question appear to be around 100-400kb each all of which appear to be in a unreadable format

I've noticed this behavior happening on my PC since the past 20'ish updates, it started just a few versions after the package installer was implemented (v185'ish?)

Can't include a log file sorry, I could not find one being created anywhere obvious

Windows 10 Pro 22H2
Defender UI1.22
i7 11700f
32GB
3060ti Dual Mini V2

**Comment by Rem0o** [comment]
Are you sure FanControl is the process creating those files?  First time hearing about this.

**Comment by FrozenApocalypse** [comment]
Yeah your correct, that was a typo on my part (my apologies) Fancontrol.exe from what I can tell isn't directly creating these Temp files. It seems svchost.exe which is being blocked for as it put it "Block abuse of in-the-wild exploited vulnerable signed drivers" yet I already have a few unsigned drivers, yet it has no problem with those

The weird part though is, Fancontrol has been given full exclusions yet this still keeps happening

I upgraded my OS to Win11 Pro 24H2 just a couple of days ago, but now the Temp files are created into Windows/SystemTemp/ which isn't even a variable of my systems default directory (as far as I am aware that doesn't even match the designated temp folder for fancontrol, right?)

Maybe this is a DefenderUI bug?

**Comment by FrozenApocalypse** [comment]
Ah sorry for that 'run around' seems it was DefenderUI that was cause of the mysterious Temp files creation...
It didn't create the exclusion list properly (a singular folder was excluded instead of the contents of that said folder)
Sorry about that Remi, great software you made there... I wish other software would work as well as yours does lol
